x-pack/plugins/ml/public/services/ml_api_service/index.js

```diff
@@ -406,5 +407,6 @@ export const ml = {
     });
   },
 
-  results
+  results,
+  jobService: jobs
```

Comment:
As discussed, in a later PR, might be good to rename this to `jobs` if the current reference to `jobs` can be renamed or moved to a separate `proxy` file.